models/user.py

```diff
@@ -56,10 +55,16 @@ class UserModel(BaseModel):
         collection_class=NobiruList,
     )
 
-    def __init__(self, **kwargs):
-        super(UserModel, self).__init__(**kwargs)
-        self.hash_digest = UserModel._hash_pw(kwargs["password"])
-        self.archived = False
+    @property
+    def password(self):
+        return self._password
```

Comment:
I think this can be removed. It's not necessary to ever access the password outside of this object.